lib/msal-common/src/authority/AuthorityOptions.ts

```diff
@@ -12,4 +12,22 @@ export type AuthorityOptions = {
     cloudDiscoveryMetadata: string;
     authorityMetadata: string;
     azureRegionConfiguration?: AzureRegionConfiguration;
+    azureCloudInstance?: AzureCloudInstance;
 };
+
+export enum AzureCloudInstance {
```

Comment:
For completeness, should PPE be added to this list?

Reply:
PPE is always internal. I do not know if we need to mix official cloud support with Msft PPE.